packages/cli/package/src/commands/local/up.ts

```diff
@@ -97,48 +95,35 @@ export default class Up extends BaseCommand<typeof Up> {
 
   async run(): Promise<void> {
     const env: FluenceEnv = "local";
-
-    try {
-      const envConfig = await initNewEnvConfig(env);
-      envConfig.fluenceEnv = env;
-      await envConfig.$commit();
-      setEnvConfig(envConfig);
-    } catch (e) {
-      this.error(
```

Comment:
We don't need catch errors here anymore?

Reply:
there previously was a check for env config that fluence.yaml exists. Now there is no such check cause e.g. providers also need env config, so there are no errors thrown there anymore, nothing to catch